code/src/components/Summary.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Summary = ({ name, activity, colour, place, temp, who }) => {
+  return (
+    <div className="content-container">
+      <p className="p-step">Complete 7 / 7</p>
+      <h4 className="sumh4">So...</h4>
+      <p>
+        {name}, picture yourself by the {place} where you are about to start your {activity}. <br />
```

Comment:
Great way of showing the results. They look so much better in a story than in a list.